pom.xml

```diff
@@ -96,22 +96,15 @@
                     <productVersion>${bamboo.version}</productVersion>
                     <productDataVersion>${bamboo.data.version}</productDataVersion>
                     <enableQuickReload>true</enableQuickReload>
-                    <enableFastdev>false</enableFastdev>
```

Comment:
FastDev is a plugin for Atlassian applications that speeds up plugin development by scanning plugin directories for changes and automatically reinstalling plugins when necessary. And QuickReload uses the same ideas as it. It is not recommended to use them at the same time.

AMPS 8.0 removed support for FastDev completely, so `<enableFastdev>false</enableFastdev>` is not necessary (and, in fact, is not supported) on AMPS 8.0+.